### PR TITLE
chore: update mermaid version to v10.9.1

### PR DESCRIPTION
**Pull request by gibso**
Hey pal,

just got some errors saying
```
Error: Evaluation failed: Error: Diagram error not found.
    at pptr://__puppeteer_evaluation_script__:45:17
    at ExecutionContext._ExecutionContext_evaluate (file:///usr/local/share/.config/yarn/global/node_modules/@mermaid-js/mermaid-cli/node_modules/puppeteer-core/lib/esm/puppeteer/common/ExecutionContext.js:254:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ExecutionContext.evaluate (file:///usr/local/share/.config/yarn/global/node_modules/@mermaid-js/mermaid-cli/node_modules/puppeteer-core/lib/esm/puppeteer/common/ExecutionContext.js:143:16)
    at async CDPJSHandle.evaluate (file:///usr/local/share/.config/yarn/global/node_modules/@mermaid-js/mermaid-cli/node_modules/puppeteer-core/lib/esm/puppeteer/common/JSHandle.js:56:16)
    at async CDPElementHandle.$eval (file:///usr/local/share/.config/yarn/global/node_modules/@mermaid-js/mermaid-cli/node_modules/puppeteer-core/lib/esm/puppeteer/common/ElementHandle.js:86:24)
    at async renderMermaid (file:///usr/local/share/.config/yarn/global/node_modules/@mermaid-js/mermaid-cli/src/index.js:241:22)
    at async parseMMD (file:///usr/local/share/.config/yarn/global/node_modules/@mermaid-js/mermaid-cli/src/index.js:213:20)
    at async run (file:///usr/local/share/.config/yarn/global/node_modules/@mermaid-js/mermaid-cli/src/index.js:508:20)
    at async cli (file:///usr/local/share/.config/yarn/global/node_modules/@mermaid-js/mermaid-cli/src/index.js:179:3)
```
when I executed asciidoctor with your image to convert mermaid diagrams. 

I came to the conclusion that this is maybe [this bug](https://github.com/mermaid-js/mermaid/issues/4140), which might have been resolved in a newer version.

Thank you!

### CODE REVIEW

**Comment by barthel** [comment]
Hi @gibso  ,

thank you for using my asciidocktor container.
Could you please open an issue for this PR?
What version of the container are you using?
Could you please provide a sample diagram that forces this error?

**Comment by gibso** [comment]
Hey @barthel, 

**TLDR: I solved it by adding double quotes to my labels. Please feel free to close this Pull Request, or, if you think an update is a good idea anyway I could create an issue here that just focuses on updating.**

while trying to prepare a minimal example that produces that error I found the actual cause.

I had a forward slash defined in the label of a diagram node. 

```
graph LR
    M[Main\n/]
```

This could be rendered successfully by other mermaid engines, like the one that renders the preview in the vscode plugin. But caused `mmdc` to crash with


```
Error: Evaluation failed: Error: Diagram error not found.
    at pptr://__puppeteer_evaluation_script__:45:17
    at ExecutionContext._ExecutionContext_evaluate (file:///usr/local/share/.config/yarn/global/node_modules/@mermaid-js/mermaid-cli/node_modules/puppeteer-core/lib/esm/puppeteer/common/ExecutionContext.js:254:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ExecutionContext.evaluate (file:///usr/local/share/.config/yarn/global/node_modules/@mermaid-js/mermaid-cli/node_modules/puppeteer-core/lib/esm/puppeteer/common/ExecutionContext.js:143:16)
    at async CDPJSHandle.evaluate (file:///usr/local/share/.config/yarn/global/node_modules/@mermaid-js/mermaid-cli/node_modules/puppeteer-core/lib/esm/puppeteer/common/JSHandle.js:56:16)
    at async CDPElementHandle.$eval (file:///usr/local/share/.config/yarn/global/node_modules/@mermaid-js/mermaid-cli/node_modules/puppeteer-core/lib/esm/puppeteer/common/ElementHandle.js:86:24)
    at async renderMermaid (file:///usr/local/share/.config/yarn/global/node_modules/@mermaid-js/mermaid-cli/src/index.js:241:22)
    at async parseMMD (file:///usr/local/share/.config/yarn/global/node_modules/@mermaid-js/mermaid-cli/src/index.js:213:20)
    at async run (file:///usr/local/share/.config/yarn/global/node_modules/@mermaid-js/mermaid-cli/src/index.js:508:20)
    at async cli (file:///usr/local/share/.config/yarn/global/node_modules/@mermaid-js/mermaid-cli/src/index.js:179:3)
```

However, as soon as I add double quotes

```
graph LR
    M["Main\n/"]
```

everything works fine. The update of mermaid I introduce in this PR here doesn't change anything. So this is rather an issue to handle in [mermaid-js/mermaid-cli](https://github.com/mermaid-js/mermaid-cli) (if at all).

Please feel free to close this Pull Request, or, if you think an update is a good idea anyway I could create an issue here that just focuses on updating.

**Comment by barthel** [comment]
Hello @gibso ,

Thank you for finding this error.
I will add a note to the example page.
I will merge the update.
Thank you again for using my project.
Please do not hesitate to create an issue/contact me if you find any other bugs or improvements.
